### DISCUSSION TITLE
How to remove the Sample Menu from the Browser example

### DISCUSSION THREAD

**Discussion by Michdo93** [eclipse-theia/theia#8998] [General] [answered]
Hi,

I am using Node.JS 12.14.1 for theia. But that is not the point. I was trying to remove the Sample Menu from the Browser example. I thought it was only one line inside `~/theia/examples/browser/src-gen/frontend/index.js` then I restarted the server. But the Menu still is shown. Okay, I thougt, then I have to rebuild it. I tried to rebuild it with `cd ~/theia && yarn rebuild:browser`. I also restarted the server. It still will not disappear.

A second point is that I changed  `~/theia/examples/browser/package.json`. And also I tried to change the title. But I will see in my browser tab still "Theia Example Browser" instead of "Theia <custom>".

How can I customize it?

Thanks in advance.

**Comment by kittaakos** [comment, marked as answer]
Just remove the extension from the `package.json` and rebuild the app:
https://github.com/eclipse-theia/theia/blob/85aa2d5dba8f9b674d896c300f7d3a36016a7e00/examples/browser/package.json#L17

> I thought it was only one line inside `~/theia/examples/browser/src-gen/frontend/index.js` then I restarted the server.

No, it will be bundled up into the `bundle.js` file. Modifying that file has no effect. Also, do not edit files under `src-gen`; they're auto-generated. You will lose your modifications when you rebuild the app.
